official/resnet/imagenet_main.py

```diff
@@ -332,6 +346,12 @@ def run_imagenet(flags_obj):
   Args:
     flags_obj: An object containing parsed flag values.
   """
+  # set the image data format that will be used by the Keras resnet model
+  if flags_obj.use_keras_model:
+    data_format = ('channels_first' if tf.test.is_built_with_cuda()
```

Comment:
Should this be `tf.test.is_built_with_cuda()` or `tf.test.is_gpu_available()`?
(The latter will return False if using the TensorFlow GPU installation but there is no GPU installed or if the GPU installed isn't "good enough")